webapp/app/partials/main/jobs.html

```diff
@@ -77,8 +77,7 @@
 </div>
 
 <div ng-show="result_sets.length == 0 && !isLoadingRsBatch.appending &&
-              !isLoadingJobs && !locationHasSearchParam('revision') &&
-              locationHasSearchParam('repo') && !currentRepo.url"
+              !isLoadingJobs && locationHasSearchParam('repo') && !currentRepo.url"
```

Comment:
Removing the |!locationHasSearchParam('revision')| is correct - great start! :-)